etl/src/main/java/org/pih/hivmigration/etl/sql/VitalsMigrator.groovy

```diff
@@ -0,0 +1,85 @@
+package org.pih.hivmigration.etl.sql
+
+class VitalsMigrator extends SqlMigrator {
+    @Override
+    def void migrate() {
+
+        executeMysql("Create vitals staging table", '''
+            CREATE TABLE hivmigration_vitals (
+              source_encounter_id int,
+              sign varchar(30),
+              result double,
+              result_unit varchar(100),
+              source_patient_id int,
+              KEY `source_encounter_idx` (`source_encounter_id`)
+            );
+        ''')
+
+        // TODO: should we do something with the 563 vitals with encounter type 'food_study'
+        //      or the 3 with type patient_contact?
+        loadFromOracleToMySql('''
+            INSERT INTO hivmigration_vitals (source_encounter_id, sign, result, result_unit, source_patient_id)
+            VALUES (?, ?, ?, ?, ?)
+        ''', '''
+            select 
+                v.ENCOUNTER_ID, 
+                v.SIGN, 
+                v.RESULT, 
+                v.RESULT_UNIT,
+                e.PATIENT_ID
+            from HIV_EXAM_VITAL_SIGNS v, HIV_ENCOUNTERS e
+            where v.ENCOUNTER_ID = e.ENCOUNTER_ID 
+            and SIGN in ('height', 'weight', 'bmi')
+            and e.TYPE in ('intake', 'followup')
+            and e.PATIENT_ID is not null
+        ''')
+
+        executeMysql("CALL create_tmp_obs_table();")
+
+        setAutoIncrement("tmp_obs", "(SELECT max(obs_id) + 1 FROM obs)")
+
+        executeMysql("Add vitals obs", '''
+            INSERT INTO tmp_obs (source_patient_id, source_encounter_id, value_numeric, concept_uuid)
+            SELECT
+                source_patient_id,
+                source_encounter_id,
+                CASE
+                    -- result unit is unreliable, but fortunately the values are unambiguous
+                    WHEN result < 3.0 THEN round(result * 100, 1)
+                    WHEN result > 40 AND result < 300 THEN result
+                END,
+                '3ce93cf2-26fe-102b-80cb-0017a47871b2'
+            FROM hivmigration_vitals
+            WHERE sign = 'height'
+            AND (result < 3.0 OR (result > 40 AND result < 300));
```

Comment:
What is this all about?  So we're arbitrarily deciding not to migrate any data if the result is in the range of [3, 40] or >= 300?  Can we remove this?